sunpy/data/_sample.py

```diff
@@ -51,52 +65,58 @@
     "NORH_LIGHTCURVE": ("tca110810", "")
 }
 
-sample_files = {}
-for key in _files:
-    sample_files[key] = os.path.abspath(os.path.join(sampledata_dir, _files[key][0]))
+
+free_required=145227776
```

Comment:
i didn't know how  to check the size of the download file so i just converted download file size to bytes. 
i will think of a different way , also if you would like to suggest me something upon it .